catalog/from-k8s/resource.go

```diff
@@ -609,3 +613,10 @@ func (t *serviceEndpointsResource) Delete(key string) error {
 	t.Service.Log.Info("delete endpoint", "key", key)
 	return nil
 }
+
+func (t *ServiceResource) serviceName(name string) string {
```

Comment:
To be just a bit more explicit about what this method does to make it easier to read in-line:

```suggestion
func (t *ServiceResource) prefixServiceName(name string) string {
```